conans/model/conan_file.py

```diff
@@ -255,6 +258,12 @@ def __init__(self, display_name=""):
         self.folders = Folders()
         self.cpp = Infos()
 
+        # LEGACY 1.X fields, can be removed in 2.X
+        self.deps_cpp_info = defaultdict(lambda: MockInfoProperty("ConanFile.deps_cpp_info"))
```

Comment:
I am not sure about these, these are not write-attributes, they are read-aggregations. I think this needs to disappear in 1.X. One thing is not wanting to break external consumers that are in 1.X, but this is being a consumer, and if it has to work in 2.0, this shouldn't be part of it.